### ISSUE TITLE
2020-01-12: Since today it doesn't work anymore

### ISSUE THREAD

**Issue by superpower10** [coldfire84/node-red-alexa-home-skill-v3-web#96]
I've been using it for months with no problems.

Since today it doesn't work anymore. I can't control any device anymore. 

I've been deleting the Alexa-Skill and all devices. Then I reinstalled the Alexa-Skill and Alexa found all my devices.

Node-Red tells me that it is connected:
<img width="655" alt="Bildschirmfoto 2020-01-12 um 10 07 15" src="https://user-images.githubusercontent.com/20533966/72216641-196ead80-3524-11ea-9e43-d424cf4a8925.png">

Node-Red-Version: RedMatic Version 5.3.9 (1.0.3: Maintenance Release)
node-red-contrib-alexa-smart-home: 0.4.66

**Comment by coldfire84** [comment]
The Dev-only version of the Alexa Skill expired last night at 23:40 UK time as it has been unchanged since the fr-FR language release. As 99% of changes happen in the Web API/ back-end I'm not surprised this has happened, but was unaware that the 90 days was coming up (had an email to say it had expired and no warning)/ there would be an associated impact.

tl;dr please check you're linked with the **production** version of the skill via the Alexa App.

**Comment by superpower10** [comment]
Thank you very much for the fast answer.

Can you please tell me, where I can find the production version?

**Comment by coldfire84** [comment]
In the Alexa App, https://red.cb-net.co.uk/docs#account-linking

**Comment by superpower10** [comment]
There are two Skills available in the Alexa App:

Node-Red and Node-RED Smart Home-Steuerung.

This morning I already installed Node-RED Smart Home-Steuerung Skill and connected to my account on https://red.cb-net.co.uk.

That's when Alexa found all my devices again but it still doesn't work.
![IMG_2223](https://user-images.githubusercontent.com/20533966/72218342-68bfd880-353a-11ea-9c7b-d8058a681a16.PNG)

![IMG_2224](https://user-images.githubusercontent.com/20533966/72218341-652c5180-353a-11ea-941f-5ba608dd0694.PNG)

**Comment by coldfire84** [comment]
> This morning I already installed Node-RED Smart Home-Steuerung Skill and connected to my account on https://red.cb-net.co.uk.

OK, so that looks right. If you join [here ](https://join.slack.com/t/cb-net/shared_invite/enQtODc1ODgzNzkxNTM3LTYwZGZmNjAxZWZmYTU4ZDllOGM3OTMxMzI4NzRlZmUzZmQ4NDljZWZiOTIwNTYzYjJmZjVlYzhhYWFiNThlMDA)you can share account username via PM so I can see whats going on/ hopefully get you backup and running.

**Comment by superpower10** [comment]
I did and sent you a message

**Comment by coldfire84** [comment]
User MQTT and web API passwords were no longer in-sync (bug with backend).
Reset passwords and then restarted Node-RED (to ensure MQTT client re-connects).
